pyvista/core/filters.py

```diff
@@ -54,6 +54,8 @@ def _get_output(algorithm, iport=0, iconnection=0, oport=0, active_scalars=None,
     data = wrap(algorithm.GetOutputDataObject(oport))
     if not isinstance(data, pyvista.MultiBlock):
         data.copy_meta_from(ido)
+        if not data.field_arrays and ido.field_arrays:
+            data.field_arrays.update(ido.field_arrays)
```

Comment:
Looks like the tests are missing coverage for L58.  Can we introduce a test for this?

Reply:
I just added a test on the `math-artist-fix_field` I thought that would push to this PR.... it has in the past 😕

Reply:
Hol up. I'm reteaching myself how to do this... I will have the cahnge on this PR momentarily

Reply:
Got it! For some reason that was more complicated than I remember it being